helpers.go

```diff
@@ -0,0 +1,49 @@
+package goStr
+
+import (
+	"strings"
+)
+
+// formatCurrency formats the currency amount with appropriate separators.
+func FormatCurrency(amountStr string, useDecimal bool, dotSeparator bool) string {
+	var formattedAmount string
+	var parts []byte
+	var separator string
+	var decimal string
+
+	if dotSeparator {
+		separator = "."
+		decimal = ","
+	} else {
+		separator = ","
+		decimal = "."
+	}
+
+	if dotSeparator && useDecimal {
+		parts = splitAmount(strings.Join(strings.Split(amountStr, "."), ","))
```

Comment:
splitAmount function only used inside this function and it has very simple logic, it would be better if u remove it. and replace this line with something like this

```go
parts = []byte(strings.Join(strings.Split(amountStr, "."), ","))
```

Reply:
done